core/storage/statistics/gae_models.py

```diff
@@ -737,3 +845,247 @@ def _get_entity_id(cls, exploration_id, exploration_version, state_name,
         return ':'.join([
             exploration_id, str(exploration_version), state_name,
             calculation_id])
+
+
+class LargeAnswerBucketModel(base_models.BaseModel):
+    """Stores answers from StateRuleAnswerLogModel for entities with very large
+    numbers of answers. Those answers are copied to this model and stored in a
+    fragmented way so that the answer migration can perform the larger
+    migrations in smaller chunks.
+    """
+    # TODO(bhenning): Remove this model once the AnswerMigrationJob has
+    # successfully run in production.
+
+    _MAX_ANSWERS_PER_BUCKET = 100
+    log_model_item_id = ndb.StringProperty(indexed=True)
```

Comment:
This isn't needed because we're removing this code once the answer migration is complete.